cmd/compute-domain-kubelet-plugin/driver.go

```diff
@@ -221,6 +221,10 @@ func (d *driver) HandleError(ctx context.Context, err error, msg string) {
 	runtime.HandleErrorWithContext(ctx, err, msg)
 }
 
+// nodePrepareResource() returns a 2-tuple; the first value is a boolean
+// indicating whether the work is 'done', the second value is a result which can
+// also reflect an error. Set the boolean to `true` for any result wrapping a
+// non-retryable error.
```

Comment:
I needed to remind myself why exactly we return `isPermanentError(err)` here below, and what the meaning of this boolean is. Maybe we want to give this a type alias in the future.